src/utils/cleanData.test.js

```diff
@@ -0,0 +1,21 @@
+import cleanData from './cleanData';
+
+test('Apples! returns apples', () => {
+  expect(cleanData('Apples!')).toBe('apples');
+});
+
+test('red pepper returns redpepper', () => {
+  expect(cleanData('red pepper')).toBe('redpepper');
+});
+
+test('B&$(#&^EaNs)))) returns beans', () => {
+  expect(cleanData('B&$(#&^EaNs))))')).toBe('beans');
+});
+
+test('SWEET  potato returns sweetpotato', () => {
+  expect(cleanData('SWEET  potato')).toBe('sweetpotato');
+});
+
+test('tofu  returns tofu', () => {
+  expect(cleanData(' tofu ')).toBe('tofu');
+});
```

Comment:
Awesome! This test is such a great clean way to verify what your expectations are into reality and the rules stay applied through the life of the app. Nicely done.

Going forward, we can identify what other functions could be tested?